docs/partials/ts-sdk-placeholder.mdx

```diff
@@ -0,0 +1,3 @@
+For developers looking to further customize the Warp UI for their own projects or build their own interchain applications, we encourage you to explore the [Typescript SDK](https://github.com/hyperlane-xyz/hyperlane-monorepo/tree/main/typescript/sdk).
+
+While we acknowledge that getting to grips with the SDK is currently a complex task, it can be a great starting point for those who are up for the challenge. We also welcome contributions and feedback from our developer community.
```

Comment:
I don't think the self-dunk is beneficial. Maybe just "The SDK contains a growing set of a utilities and abstractions for interacting with Hyperlane across different protocols (EVM, Cosmos, Solana)."